triggers/daml/Daml/Trigger/Internal.daml

```diff
@@ -45,15 +53,53 @@ newtype TriggerA a = TriggerA (State TriggerAState a)
 addCommands : Map CommandId [Command] -> Commands -> Map CommandId [Command]
 addCommands m (Commands cid cmds) = Map.insert cid cmds m
 
+#ifdef DAML_GENMAP && DAML_GENERIC_COMPARISON
+adjust : forall k v. Ord k => k -> (Optional v -> Optional v) -> GMap.Map k v -> GMap.Map k v
```

Comment:
@sofiafaro-da perhaps something worth adding to `DA.Map`? What do you think? (not in this PR)

Reply:
yeah, that looks like a good function to have in DA.Map :)

Reply:
#7427